### ISSUE TITLE
Loss doesn't go down

### ISSUE THREAD

**Issue by darsh10** [facebookresearch/detr#311]
Training on a custom dataset. However, the loss doesn't seem to go below 14. This is rather surprising, even with very few images. Even with 100 images only, I face the same issue. Any possible solution?

python -u main.py --coco_path ../stitchingdataset/ --batch_size 2 --num_queries 204 --lr 0.00001

**Comment by alcinos** [comment]
The total loss is an aggregate of many things, it would probably be more helpful to check which of the losses isn't converging to 0. Intrinsically, the intermediate losses may never converge to 0 because the first few layers may not have the capacity to fit the data.
Also, even with few datapoints, the data augmentation will probably prevent a perfect fit

I recommend looking more in details the behavior of each loss, and start by overfitting on a single batch with no data-augmentation, and build from there.
Best of luck

**Comment by jinweiLiu** [comment]
i met the same problem when training on luna data

**Comment by darsh10** [comment]
Were you able to fix it ?

**Comment by jinweiLiu** [comment]
no, I just want to ask that if anybody fixed this problem

**Comment by lucastononrodrigues** [comment]
I finetuned it with bdd100k and managed to get the loss down from 60 to around 5 in 20 epochs.
